templates/ingress.yaml

```diff
@@ -30,3 +30,36 @@ spec:
           serviceName: neuvector-service-webui
           servicePort: 8443
 {{- end }}
+---
+{{- if and .Values.controller.enabled .Values.controller.ingress.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: neuvector-restapi-ingress
+  namespace: {{ .Release.Namespace }}
+{{- with .Values.controller.ingress.annotations }}
+  annotations:
+{{ toYaml . | indent 4 }}
+{{- end }}
+  labels:
+    chart: {{ template "neuvector.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+{{- if .Values.controller.ingress.tls }}
+  tls:
+  - hosts:
+    - {{ .Values.controller.ingress.host }}
+{{- if .Values.controller.ingress.secretName }}
+    secretName: {{ .Values.controller.ingress.secretName }}
+{{- end }}
+{{- end }}
+  rules:
+  - host: {{ .Values.controller.ingress.host }}
+    http:
+      paths:
+      - path: {{ .Values.controller.ingress.path }}
+        backend:
+          serviceName: neuvector-service-restapi
```

Comment:
am I missing something? I can't see the service for this

Reply:
That is in controller-service.yaml. https://github.com/hmcts/neuvector-helm/pull/1/files#diff-dace72095c9160121bd33265cd9dfafa